### PR TITLE
include the upstream version number in the `--version` output

### LINKED ISSUE #92: Say what pyright it's based on
Maybe in the `--version` or something

related: #132

### LINKED ISSUE #132: rework the versioning
currently i use lerna to increment the version, but that always causes needless upstream conflicts. all the node packages are private in basedpyright and the only public facing package is the pypi one, so there should be no need to touch the metadata on those packages at all

### PR DESCRIPTION
**Pull request by DetachHead**
- fixes #92 
- fixes #132

### CODE REVIEW

File: packages/pyright-internal/src/version.json [hunk 1]
```diff
@@ -0,0 +1 @@
+"1.7.0"
```
Comment (KotlinIsland) [inline packages/pyright-internal/src/version.json:1]: Lets see a cli test case that checks the version output

Comment (DetachHead) [reply packages/pyright-internal/src/version.json:1]: sounds like a pain in the ass, the code for it is just 2 console.logs, very straightforward and unlikely to break imo

**Comment by AThePeanut4** [comment]
This only seems to work when building from source, the release packages print the incorrect upstream version:
```
> basedpyright --version
basedpyright 1.8.0
based on pyright 1.8.0
```
This happens when installing with `pip install basedpyright`, and with `npm install -g basedpyright` (the latter installs as `pyright`, so the command there is `pyright --version`).

**Comment by DetachHead** [comment]
thanks, i raised a new issue for that (#251). will try to fix it for today's release

**Pull request merged**